packages/block-library/src/image/utils.js

```diff
@@ -3,6 +3,15 @@
  */
 import { NEW_TAB_REL } from './constants';
 
+export function evalAspectRatio( value ) {
```

Comment:
```suggestion
export function calculateAspectRatio( value ) {
```

Nit: the word "eval" in JavaScript was a trigger for me 😄 I'm not sure we're evaluating here anyway - more "calculating".

Reply:
I guess I chose `eval` because it's doing the same thing as the JS `eval()` taking some source code string (in our case a CSS aspect-ratio value) and returning the numerical aspect ratio. `eval('5/8')`, for example, does the same thing.

I'd expect something with `calculate` in the name to take a numerical `width` and `height` argument instead of a single string.

Reply:
Ah. You're right. It's a string. I missed that. Moving too quickly 👍